src/packages/@app/pages/Teams/Teams.ViewModel.ts

```diff
@@ -135,12 +137,22 @@ export class TeamsViewModel {
     await this.workspaceRepository.setActiveWorkspace(id);
     navigate("/dashboard/collections");
   };
+
+ /**
+   * Switch from one team to another
+   * @param id - Workspace id
+   */
```

Comment:
I think id should be team id here. not workspace id

Reply:
Yes true